src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -420,6 +426,12 @@ doclet.usage.windowtitle.parameters=\
 doclet.usage.windowtitle.description=\
     Browser window title for the documentation
 
+doclet.usage.date.parameters=\
+    <date-and-time>
+doclet.usage.date.description=\
+    Specifies the value to be used to timestamp the generated\n\
+    pages, in ISO_ZONED_DATE_TIME format
```

Comment:
I guess it is technically correct to use  a `DateTimeFormatter` constant name here, but wouldn't it be better form to use a generic term like "ISO-8601 format"?

Reply:
Yeah, you're right, that would be better, and sufficient.

Reply:
Fixed